GitVersion.yml

```diff
@@ -1 +1 @@
-mode: Mainline
\ No newline at end of file
+mode: ContinuousDeployment
```

Comment:
I know the package does not align very well with Cake's version (something we should likely change too). Can you elaborate on why this changes? IIRC, the pipeline simply publishes every commit, right?

Reply:
Exactly. 
Mainline is not supported anymore with GitVersion v6.